src/main/java/car/domain/Cut.java

```diff
@@ -1,11 +1,12 @@
-package car;
+package car.domain;
 
 public class Cut {
+    private static final int NAME_MAX_SIZE = 5;
 
     public Car getCar(String carName) {
         int length = carName.length();
-        if (length > 5) {
-            length = 5;
+        if (length > NAME_MAX_SIZE) {
+            length = NAME_MAX_SIZE;
         }
         Car car = new Car(carName.substring(0, length), 0);
         return car;
```

Comment:
```suggestion
        return new Car(carName.substring(0, length), 0);
```
변수에 담아서 다른 곳에서 사용하지 않으면 리턴을 통해 바로 처리할 수 있겠죠?